spec/dummy/app/assets/stylesheets/active_admin.scss

```diff
@@ -27,3 +27,15 @@ $approved-color: #08A510;
   &.rejected { background: $rejected-color; }
   &.approved { background: $approved-color; }
 }
+
+.bool_value {
+  font-size: 12px;
+
+  &.true_value {
+    color: green;
```

Comment:
Color `green` should be written in hexadecimal form as `#008000`<br>Color literals like `green` should only be used in variable declarations; they should be referred to via variable everywhere else.